tasks/src/automatic-speech-recognition/about.md

```diff
@@ -46,4 +66,6 @@ These events help democratize ASR for all languages, including low-resource lang
 ## Useful Resources
 - [Making automatic speech recognition work on large files with Wav2Vec2 in 🤗 Transformers](https://huggingface.co/blog/asr-chunking)
 - [Boosting Wav2Vec2 with n-grams in 🤗 Transformers](https://huggingface.co/blog/wav2vec2-with-ngram)
-- [ML for Audio Study Group - Intro to Audio and ASR Deep Dive](https://www.youtube.com/watch?v=D-MH6YjuIlE)
\ No newline at end of file
+- [ML for Audio Study Group - Intro to Audio and ASR Deep Dive](https://www.youtube.com/watch?v=D-MH6YjuIlE)
+- [Massively Multilingual ASR: 50 Languages, 1 Model, 1 Billion Parameters](https://arxiv.org/pdf/2007.03001.pdf)
+- [NeMo ToolKits](https://github.com/NVIDIA/NeMo)
```

Comment:
Can you add what it does 😅 I know nemo but a beginner doesn't, it can be something along the lines of:
An ASR Toolkit made by NVIDIA: NeMo

Reply:
Yeah, I will add more info about NeMo!